apps/explorer/src/components/AssetLogo.tsx

```diff
@@ -31,14 +35,24 @@ interface AssetLogoProps {
   className?: string
 }
 
+// TODO: Use type assertion from tanstack work in DW
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const isNFTMetadataWithFile = (metadata: any): metadata is NFTMetadataWithFile => 'file' in metadata
+
 const AssetLogo = (props: AssetLogoProps) => {
   const { assetId, showTooltip, className } = props
 
   const theme = useTheme()
   const metadata = useAssetMetadata(assetId)
+  const NFTImage = isNFTMetadataWithFile(metadata) ? metadata.file.image : undefined
```

Comment:
nit: shouldn't start with capital if it's not a component. How about `nftImageSrc`?

Reply:
You're right. I always have this reflex of using a capital N for NFT :p

Reply:
Done